shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformViewsChannel.java

```diff
@@ -36,6 +38,13 @@ public void invokeViewFocused(int viewId) {
     channel.invokeMethod("viewFocused", viewId);
   }
 
+  private String detailedExceptionString(Exception e) {
```

Comment:
Since this method takes a logging action, rather than just creating a "detailed exception string", would you mind renaming to something that mentions the action it takes? Also, it doesn't seem to be adding any detail, so I'm a bit confused as to what makes this a "detailed exception" vs a normal exception. Maybe `logException(e)` or `logStacktrace(e)`?

Reply:
It's not logging anything... just builds a string

Reply:
Oh, I mis-red the `printStackTrace` line. Can we still rename the method to simply refer to a stack trace rather than detailed exception? I assume that this method isn't intended to be overloaded for other purposes, right? If not, then using "stacktrace" will be unambiguous to developers, whereas "detailed exception" could mean a number of things.

Also, a nit, I think we have a policy against one letter variables for errors/exceptions. If I'm remembering that correctly then you may want to adjust the input from `e` to `exception` or something similar.

Reply:
Renamed to `e` to `exception`.
Also made the method static.

Regarding the name, note that i actually prints the exception message and the stack trace (initially I actually had it concatenate those, but turns out `printStackTrace` already prepends the exception's toString).

Reply:
SG